.github/workflows/sycl-linux-precommit.yml

```diff
@@ -94,6 +94,13 @@ jobs:
             reset_gpu: true
             install_drivers: ${{ contains(needs.detect_changes.outputs.filters, 'drivers') }}
             extra_lit_opts: --param gpu-intel-gen12=True
+          - name: E2E tests on Intel Ponte Vecchio GPU
```

Comment:
Is it possible to also add the task for PVC, similar to name: "E2E tests with **dev igc** on Intel Arc A-Series Graphics"?

Reply:
Yes, we can run E2E tests on PVC with dev igc as well. My only concern is that currently we have only one PVC runner - the second runner is MIA - and we are using it for build as well as to run E2E tests. I'm afraid that adding another PVC task with dev igc might cause the PVC runner to become a bottleneck in the CI pipeline.

How about I create a GH issue to run E2E tests on PVC using dev igc for now? and when the second PVC runner is back in intel/llvm CI, I can enable the "dev igc" tests as well.

@sarnex what do you think?

Reply:
After some offline discussion, it turns out that we can actually enable running E2E tests with dev igc on PVC, without it becoming a bottleneck: 
Running E2E tests on PVC takes ~5min while running E2E tests on Gen12 takes ~50min (https://github.com/intel/llvm/actions/runs/10115683794/job/27977273715) 😱

Reply:
good news! :)